tests/features/monrun.feature

```diff
@@ -513,3 +513,14 @@ Feature: ch-monitoring tool
     """
     2;ERROR
     """
+  
+  Scenario: Check clickhouse orphaned objects with long error_msg
+    When we create file /tmp/object_storage_cleanup_state.json with data "{ \"orphaned_objects_size\": 0,  \"error_msg\": \"Code: 27. DB::Exception: Cannot parse: input:: expected '\\\\t' before: 'klg%2D1acvr8hmq0n16qm5%2Edb%2Eyandex%2Enet\\\\ndefault\\\\n6736d483-516a-4892-87d4-084d5c1f6d3c\\\\n': While executing SystemRemoteDataPaths. (CANNOT_PARSE_INPUT_ASSERTION_FAILED) (version 24.8.5.115 (official build))  Query: SELECT obj_path, obj_size FROM _system.listing_objects_from_object_storage AS object_storage LEFT ANTI JOIN remoteSecure('klg-1acvr8hmq0n16qm5.db.yandex.net', system.remote_data_paths) AS object_table ON object_table.remote_path = object_storage.obj_path AND object_table.disk_name = 'object_storage' SETTINGS traverse_shadow_remote_data_paths=1 FORMAT TabSeparated (klg-1acvr8hmq0n16qm5.mdb.yandex.net)\" }"
+    And we execute command on clickhouse01
+    """
+    ch-monitoring orphaned-objects --state-local
+    """
+    Then we get response
+    """
+    2;Code: 27. DB::Exception: ... (CANNOT_PARSE_INPUT_ASSERTION_FAILED) (version 24.8.5.115 (official build)) ...
```

Comment:
Let's use `we get response matches` for checking response without specific versions

Reply:
The purpose of this test is to show the final error message, we are parsing an error that already contains "(version 24.8.5.115 (official build))", that is, this result of this test does not depend on the version of clickhouse.

Maybe I didn’t understand why to change this test.

Reply:
Ok, I got confused with the `Scenario: Check Geobase` test. But it is already fixed